src/components/layouts/AdminLayout/AdminLayout.tsx

```diff
@@ -25,32 +30,39 @@ export const AdminLayout: React.FC<AdminLayoutPropsType> = ({ children }) => {
 
   const [pageLayout, setPageLayout] = useState({ sidebarOpen: false });
 
-  const setSidebarOpen = useMemo(
-    () => (newValue: boolean) => setPageLayout({ sidebarOpen: newValue }),
-    [setPageLayout]
+  const closeSidebar = useCallback(
+    () => setPageLayout({ sidebarOpen: false }),
+    []
   );
 
+  const triggerSidebar = useCallback(() => {
+    setPageLayout({ sidebarOpen: !pageLayout.sidebarOpen });
+  }, [pageLayout.sidebarOpen]);
+
   return (
-    <div>
+    <div className="AdminLayout">
       <AdminNavBar
         sidebarOpen={pageLayout.sidebarOpen}
-        setSidebarOpen={setSidebarOpen}
+        onCloseSidebar={closeSidebar}
       />
       <div className="md:pl-56 flex flex-col flex-1">
         <div className="sticky top-0 z-10 md:hidden pl-1 pt-1 sm:pl-3 sm:pt-3 bg-gray-100">
           <button
             type="button"
             className="-ml-0.5 -mt-0.5 h-12 w-12 inline-flex items-center justify-center rounded-md text-gray-500 hover:text-gray-900 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500"
```

Comment:
[NOTE]
I guess if there are some really long sausages, a constant can be used to improve readability, reduce horizontal scroll.

Just a thought, not that important in this case

Reply:
moved to a const upper in the file: https://github.com/sparkletown/sparkle/pull/2812/commits/0f78efe3a381196c9e5b7cacf68aa5c14daa88bc